ethereumj-core/src/main/java/org/ethereum/config/blockchain/Eip160HFConfig.java

```diff
@@ -76,8 +76,7 @@ public Constants getConstants() {
     @Override
     public boolean acceptTransactionSignature(Transaction tx) {
         // Restoring old logic. Making this through inheritance stinks too much
-        if (!tx.getSignature().validateComponents() ||
-                tx.getSignature().s.compareTo(SECP256K1N_HALF) > 0) return false;
-        return  tx.getChainId() == null || Objects.equals(getChainId(), tx.getChainId());
+        return tx.getSignature().validateComponents() &&
```

Comment:
Hi @nicksavers , Can you please point me to the test class to be updated for this?
I can see 100% coverage in coveralls for `Eip160HFConfig.java`, https://coveralls.io/builds/12636785